lib/WebService/PayPal/PaymentsAdvanced/Role/HasCreditCard.pm

```diff
@@ -23,9 +22,8 @@ has card_expiration => (
 
 has card_last_four_digits => (
     is       => 'lazy',
-    isa      => PositiveInt,
+    isa      => SimpleStr,
```

Comment:
Did `PositiveOrZeroInt` not work for this fix?

Reply:
I think that type would work too. That was initially what I was going to do. However it would also allow things like "00000" and "0000123" I believe. As such, I think that treating this as a string is more semantically accurate.